restapi.go

```diff
@@ -1332,23 +1324,7 @@ func (s *Session) ChannelMessageSendEmbed(channelID string, embed *MessageEmbed)
 // content   : The message to send.
 // embed     : The embed data to send.
 func (s *Session) ChannelMessageSendEmbedWithMessage(channelID string, content string, embed *MessageEmbed) (st *Message, err error) {
```

Comment:
Good call, let's kill this method, it was added in develop, I don't care too much to keep it around.